lib/twitter/twits.ex

```diff
@@ -0,0 +1,163 @@
+defmodule Twitter.Twits do
+  @moduledoc """
+  The Twits context.
+  Includes functions for posts and likes
+  """
+
+  import Ecto.Query, warn: false
+  alias Twitter.Repo
+  alias Twitter.Twits.Post
+  alias Twitter.Twits.Like
+  alias Twitter.Accounts
+
+  defp user_posts_query(query, %Accounts.User{id: user_id}) do
+    from p in query, where: p.user_id == ^user_id
+  end
+
+  @spec list_posts(User.t()) :: list(Post.t())
+  @doc """
+  Returns the list of posts. This fn is used for the current user.
+  ## Examples
+      iex> list_posts()
+      [%Post{}, ...]
+  """
+  def list_posts(%Accounts.User{} = user) do
```

Comment:
[nit]: can you move spec to directly above function to maintain consistent style

Reply:
Is there a way for your user to see all posts by all users or is it intentionally limited to their own tweets?

Reply:
a User can see and manipulate their own posts through PostController. a User can see other user's posts (all at ones) through BrowseController

Reply:
Ok, I have moved the spec around in other places too.